ultradns-rest/src/main/java/denominator/ultradns/NetworkConnection.java

```diff
@@ -0,0 +1,29 @@
+package denominator.ultradns;
+
+import javax.inject.Inject;
+
+import denominator.CheckConnection;
+
+class NetworkConnection implements CheckConnection {
+
+  private final UltraDNSRest api;
+
+  @Inject
+  NetworkConnection(UltraDNSRest api) {
+    this.api = api;
+  }
+
+  @Override
+  public boolean ok() {
+    try {
+      return "GOOD".equals(api.getNeustarNetworkStatus().getMessage().toUpperCase());
```

Comment:
What if getMessage() returns null? Better to use equalsIgnorecase and not call toUpperCase on getMessage.